<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeomFilterGroup</class>
 <widget class="QDialog" name="GeomFilterGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Group for same criteria</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gb_ref_g">
     <property name="title">
      <string>Reference Group:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,1">
      <item row="0" column="0">
       <widget class="QPushButton" name="pb_ref_g">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_ref_g"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="l_nam_g">
        <property name="text">
         <string>Name result Group:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="le_nam_g">
        <property name="text">
         <string>Group_R</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="cb_size">
     <property name="title">
      <string>Criteria</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,1">
      <item row="0" column="0" rowspan="5">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>134</width>
          <height>138</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="cb_size_2">
        <property name="text">
         <string>Size</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="cb_locx">
        <property name="text">
         <string>Location X</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="cb_locy">
        <property name="text">
         <string>Location Y</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="cb_locz">
        <property name="text">
         <string>Location Z</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="cb_norm">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Normal</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_tol">
     <property name="title">
      <string>% Tolerance</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>134</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="sb_tol">
        <property name="minimum">
         <double>0.010000000000000</double>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GeomFilterGroup</receiver>
   <slot>proceed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GeomFilterGroup</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_ref_g</sender>
   <signal>clicked()</signal>
   <receiver>GeomFilterGroup</receiver>
   <slot>selectGroupRef()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>proceed()</slot>
  <slot>selectGroupRef()</slot>
 </slots>
</ui>
